### ISSUE TITLE
Start visual development

### ISSUE THREAD

**Issue by LeoJhonSong** [TDPS-Mihotel/Mihotel#16]
@TDPS-Mihotel/Visual Please create your own repo **from your team's page** and start to look for method to reach your tasks.

:bulb: since there are quite a lot of tools and method to deal with images, I suggest you **just browse first** and update useful reference to your repo to share ideas with each other (or simply chat by QQ)

**Comment by LeoJhonSong** [comment]
Thess may help you:
- [图像的四种类型及简述](https://blog.csdn.net/r_w_zhang/article/details/78475812)
- [Python-opencv 图片颜色域的识别选取](https://blog.csdn.net/sevensevensevenday/article/details/62418959?locationNum=5&fps=1)
- [lena图由来](https://blog.csdn.net/xiaosongaa/article/details/68925639)
- [图像二值化: 大津算法](https://www.cnblogs.com/gezhuangzhuang/p/10295181.html)
- [图像形态学常见处理](https://blog.csdn.net/sunny2038/article/details/9137759)
- [OpenCV形状识别](https://www.pyimagesearch.com/2016/02/08/opencv-shape-detection/)
- [图像常用特征介绍](https://wenku.baidu.com/view/d0275804ad51f01dc381f11a.html)
- [给深度学习入门者的Python快速教程 - 番外篇之Python-OpenCV](https://zhuanlan.zhihu.com/p/24425116)
- [python + openCV 实现图像轮廓识别和面积计算](https://blog.csdn.net/whymeyan/article/details/78856964)
- [Opencv3.0-python的那些事儿：（一）、Opencv的图像和视频处理基本用法](http://www.itkeyword.com/doc/5142059619854589x334/python3.4-opencv-opencv3.0)
- [相机标定究竟在标定什么？
许翔翔](https://zhuanlan.zhihu.com/p/30813733)
- [树莓派上利用Tensorflow实现小车的自动驾驶](https://my.oschina.net/u/3858986/blog/2875027)
- [3D可视化CNN手写数字识别演示](https://www.cs.ryerson.ca/~aharley/vis/conv/)

**Comment by shuchang** [comment]
Some DL methods to do path recognition:
* Supervised Learning:
  * [Marking the center point](https://becominghuman.ai/autonomous-racing-robot-with-an-arduino-a-raspberry-pi-and-a-pi-camera-3e72819e1e63) ([中文翻译](https://cloud.tencent.com/developer/news/73131))
  * [Reference 1 for some state of the art](https://zhuanlan.zhihu.com/p/37801090)
  * [Reference 2 for some state of the art](https://www.jiqizhixin.com/articles/2018-06-04-17)

**Strengths**: Given the labels, we can include images in extreme environmental situations to enhance the robustness of the trained model
**Weaknesses**: We need to give labels to the training sets manually

* Unsupervised Learning:
  [Unsupervised-Segmentation](https://kanezaki.github.io/pytorch-unsupervised-segmentation/ICASSP2018_kanezaki.pdf) ([中文翻译](https://zhuanlan.zhihu.com/p/68528056))
  [FCAN](https://arxiv.org/abs/1804.08286) ([中文翻译](https://blog.csdn.net/longxinghaofeng/article/details/83993614))
  [Double DIP](https://arxiv.org/abs/1812.00467) ([中文翻译](https://www.bilibili.com/read/cv2802966/))

**Strengths**: No need for labeling
**Weaknesses**: To obtain good clustering results during training, photos should have clear boundaries. However, during the test, the quality of photos might suffer from environmental factors, such as sunlight, shadow and so on

In general, we hope to use the DL methods to achieve two goals:
* Obtain a trained model for the path recognition
* Learn features that traditional image processing methods fail to learn
